docs/managing-dragonfly/managing-dragonfly.md

```diff
@@ -3,3 +3,14 @@ sidebar_position: 5
 ---
 
 # Managing Dragonfly
+
+### Limits
+
+Listed below are known Dragonfly limitations.
+
+* Each Dragonfly instance can handle up to two billion keys
```

Comment:
Also not true, the cursor is
```
uint32_t seg_id_;
uint8_t bucket_id_;
uint8_t slot_id_;
```
and (almost) everywhere size_t is used, but I doubt again it will be ever used with so many keys